content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title:
+  'Making observability fun: How we increased engineers confidence in incident
+  management using a game'
+linkTitle: Skyscanner using OtelDemo
```

Comment:
(nit) I think it's normally spelt OTel Demo :)

Reply:
```suggestion
linkTitle: Skyscanner using OTel Demo
```